crates/git_ui/src/git_panel.rs

```diff
@@ -324,6 +324,7 @@ pub struct GitPanel {
     context_menu: Option<(Entity<ContextMenu>, Point<Pixels>, Subscription)>,
     modal_open: bool,
     _settings_subscription: Subscription,
+    pub(crate) amend_commit: bool,
```

Comment:
I'm not sure what the guideline is about public properties; this is currently needed so that the `CommitModal` can read it, so that both checkboxes are in sync.